### PR TITLE
Lesson 02 files; series.py now updated with function lucas.

### PR DESCRIPTION
**Pull request by duanez2021**
Lesson 2 files. Grid Printer parts 2 and 3 in separate files; fizzbizz; series.py is still in work as it needs the Lucas numbers but i wanted to make a merge now to learn how.

### CODE REVIEW

File: students/duanez2021/lesson02/fizzBizz.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#  https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/fizz_buzz.html
+#
+#  Write a program that prints the numbers from 1 to 100 inclusive.
+#  But for multiples of three print “Fizz” instead of the number.
+#  For the multiples of five print “Buzz” instead of the number.
+#  For numbers which are multiples of both three and five print “FizzBuzz” instead.
+
+##############################################################
+
+def fizzBuzz():
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/fizzBizz.py:16]: In Python function names (and almost everything else besides classes) should be snake_case.   So, fizz_buzz():

File: students/duanez2021/lesson02/fizzBizz.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    n= 101
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/fizzBizz.py:17]: Need a whitespace before assignment operator

File: students/duanez2021/lesson02/fizzBizz.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    for i in range (n):
+        if i != 0:
+            if (i % 3 == 0) and (i % 5 == 0):
+                print('FizzBuzz')
+            elif (i % 3 == 0) and (i % 5 != 0):
+                print('Fizz')
+            elif  (i % 3 != 0) and (i % 5 == 0):
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/fizzBizz.py:24]: Too many spaces after elif

File: students/duanez2021/lesson02/fizzBizz.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+                print('Buzz')
+            else: print(str(i))
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/fizzBizz.py:26]: You'll want to drop the print to the next line

File: students/duanez2021/lesson02/fizzBizz.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        else:
+            None
+
+fizzBuzz()
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/fizzBizz.py:30]: Works well!

File: students/duanez2021/lesson02/gridPrinter_part2.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+# then use those two functions in a print_grid function
+##############################################################
+
+def print_row(n):
+    #print a row of +/-
+    plus = '+'
+    minus = '-'
+    row = ''
+    # if n < 3 default to 3
+    if n < 3: n = 3
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part2.py:21]: You'll want this on two lines

File: students/duanez2021/lesson02/gridPrinter_part2.py [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+        i = i + 1
+        if i <= 3:
+            for y in range(dash):
+                row = row + minus
+    return row
+
+def print_vRow(n):
+    vLine = '|'
+    vRow = ''
+    if n < 3: n = 3
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part2.py:36]: This should be two lines

File: students/duanez2021/lesson02/gridPrinter_part2.py [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        i = i + 1
+        if i <= 3:
+            for y in range(dash):
+                vRow = vRow + ' '
+    return vRow
+
+def print_grid(n):
+    box = ''
+
+    if n < 3: n = 3
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part2.py:51]: Two lines here as well.

File: students/duanez2021/lesson02/gridPrinter_part3.py [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+# then use those two functions in a print_grid function
+##############################################################
+
+def print_row2(column_size, cell_size):
+    #print a row of +/-
+    plus = '+'
+    minus = '-'
+    row = ''
+    # if n < 3 default to 3
+    if cell_size < 1: cell_size = 1
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part3.py:21]: This should be two lines, here and elsewhere with any if statement

File: students/duanez2021/lesson02/gridPrinter_part3.py [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+        if i <= column_size+1:
+            for y in range(cell_size):
+                row = row + minus
+    return row
+#print (print_row2(1, 1))
+
+def print_vRow2(col_size, cell_size):
+    vLine = '|'
+    vRow = ''
+    if cell_size < 1: cell_size = 1
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part3.py:38]: Missing whitespace around colon, here as well as below

File: students/duanez2021/lesson02/gridPrinter_part3.py [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+                vRow = vRow + ' '
+    return vRow
+
+#print(print_vRow2(1,1))
+
+
+def print_grid2(grid_size, cell_size):
+    box = ''
+
+    if grid_size <= 1:  grid_size=2
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part3.py:56]: Missing whitespace around assignment operator - careful to be consistent

File: students/duanez2021/lesson02/gridPrinter_part3.py [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+    if cell_size <= 1:  cell_size = 1
+
+    c = 1
+
+    for i in range(grid_size+1):
+        box = box + print_row2(grid_size, cell_size) + '\n'
+        c = c + 1
+        if c <=grid_size + 1:
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part3.py:64]: Missing whitespace here too

File: students/duanez2021/lesson02/gridPrinter_part3.py [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+            for j in range(cell_size):
+                box = box + print_vRow2(grid_size, cell_size) + '\n'
+    return box
+
+print(print_grid2(3,2))
```
Comment (charlesrobison) [inline students/duanez2021/lesson02/gridPrinter_part3.py:69]: Shouldn't need to print a function

**Comment by duanez2021** [comment]
Thank you. I will take a look do another push for the series.py file.

**Comment by duanez2021** [comment]
Hi! I did a PR yesterday with a revised mailroom (mailroom_rev01.py) and the Lab Series file (List_Lab_Series_01_Rev03.py). I split the list lab into four separate files originally.

The mailroom module appears to work much better now. Take a look!

**Pull request merged**